src/main/java/de/eldoria/eldoutilities/scheduling/QueuingSelfSchedulingTask.java

```diff
@@ -94,4 +97,9 @@ protected boolean remove(T o) {
     protected boolean removeIf(Predicate<? super T> filter) {
         return tasks.removeIf(filter);
     }
+
+    protected void clear() {
```

Comment:
Shouldn't the method actually be public and receive a small javadoc?
Clearing the task only from inside sounds not much useful

Reply:
> Clearing the task only from inside sounds not much useful
Do you think it would be wise to give it another behavior? I was thinking about such a method because some remaining tasks don't have to be executed with ``#shutdown`` in Landlord.

Reply:
Since this uses the bukkit scheduler in the background with a repeating task all tasks will be discarded when the server stops or when the bukkit scheduler stops which should be the point when the plugins get disabled iirc. Do you have another experiences there?

I thought of this actually more like a general method if you want to stop immediately when you have submitted a lot of tasks or in case you want to force a clean shutdown. Both would actually require the method to be public

Reply:
``#shutdown`` method probably does the job for a clean shutdown, I'm gonna clarify the added method to simply clear and kill the current scheduler _(stop immediately as you wrote)_.